crawler/cmd/one.go

```diff
@@ -59,25 +65,22 @@ func getPAfromWhiteList(repoURL string, args []string) (pa crawler.PA) {
 		// looking into repositories
 		for _, paWlRepo := range paWl.Repositories {
 			log.Tracef("matching %s with %s", paWlRepo, repoURL)
-			if paWlRepo == repoURL {
+			if (url.URL)(paWlRepo) == repoURL {
 				log.Debugf("PA found in whitelist %+v", paWl)
 				return paWl
 			}
 		}
 		// looking into organizations
 		for _, paWlRepo := range paWl.Organizations {
 			log.Tracef("matching %s.* with %s", paWlRepo, repoURL)
```

Comment:
**[error]** pawlRepo and repoURL are url.URL now, not string

Reply:
Thanks, fixed in the other branch :+1:

Reply:
Cool!